backend/src/main/webapp/blackswan/script.js

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const FLAG_LOCAL = 1;
+/** 
+ * When testing locally, need to append host that python service 
+ * runs on to requests.
+ */
+const LOCAL = FLAG_LOCAL === 1 ? 'http://localhost:8889' : '';
+
+/** Send GET request to trigger cron job to run. */
+const runJob = async () => {
+  const response = await fetch('/blackswan/test');
+  const message = await response.text();
+  console.log("job ran: " + message);
+}
+
+/** Send GET request to load updated data into Cloud Storage. */
+const loadData = async (time) => {
```

Comment:
is there a reason why this variable is called time?

Reply:
i called it `time` because the input (based on what button is clicked) will determine the time range of the data loaded into cloud storage!